docker.mk

```diff
@@ -0,0 +1,62 @@
+include .env
+
+.PHONY: up down stop prune ps shell wp logs
+
+default: up
+
+WP_ROOT ?= /var/www/html/
+
+## help	:	Print commands help.
+help : docker.mk
+	@sed -n 's/^##//p' $<
+
+## up	:	Start up containers.
+up:
+	@echo "Starting up containers for for $(PROJECT_NAME)..."
```

Comment:
I don't know if this is a typo but **for** appear twice.

Reply:
Now tracked in wodby/docker4wordpress#50.